cmd/flux/test.go

```diff
@@ -52,26 +52,37 @@ func (testExecutor) Run(pkg *ast.Package) error {
 
 	var output strings.Builder
```

Comment:
Maybe change the name of this variable to reflect that it can only contain output from `diff`?

Reply:
If we add `testing.shouldError` the output could contain something else than a diff, so I think that may be too specialized.